relays/bin-substrate/src/cli/register_parachain.rs

```diff
@@ -231,22 +223,18 @@ impl RegisterParachain {
 			.into();
 			let force_lease_signer = relay_sign.clone();
 			relay_client
-				.submit_signed_extrinsic(relay_sudo_account.clone(), move |_, transaction_nonce| {
-					Ok(Bytes(
-						Relaychain::sign_transaction(SignParam {
-							spec_version,
-							transaction_version,
-							genesis_hash: relay_genesis_hash,
-							signer: force_lease_signer,
-							era: relay_substrate_client::TransactionEra::immortal(),
-							unsigned: UnsignedTransaction::new(
-								force_lease_call.into(),
-								transaction_nonce,
-							),
-						})?
-						.encode(),
-					))
-				})
+				.submit_signed_extrinsic(
+					relay_sudo_account.clone(),
```

Comment:
@svyatonik 
I guess this last `relay_sudo_account.clone()` could be just `relay_sudo_account`